.gitlab-ci.yml

```diff
@@ -69,20 +72,49 @@ deploy:
   <<: *deploy-common
   only:
     - tags
+  except:
+    - schedules
   
   
-deploy (ad hoc): &deploy
-  <<: *deploy-common
-  when: manual
+# deploy (ad hoc): &deploy
+#   <<: *deploy-common
+#   when: manual
+
```

Comment:
I've also ad-hoc deployment for now. I'll bring that back once we have a staging env and then I'll enable it there (and only there).